### ISSUE TITLE
Representação de passagens “aéreas” de ligação aos edificios

### ISSUE THREAD

**Issue by CatarinaGaspar** [dgterritorio/RECART#166]
Boa Tarde, 

Estas passagens são representáveis? Se sim, como devem ficar classificadas?

![image](https://user-images.githubusercontent.com/60113889/220407784-8966c9bf-57c1-42dc-9b85-e1cd353f3a8a.png)
![image](https://user-images.githubusercontent.com/60113889/220407798-e677cef4-5ea5-40f2-a24e-7cd5333e711b.png)

Obrigado,

**Comment by aserronha** [comment]
Boa tarde,

Estas situações enquadram-se no valor "Associado à Residência". Ver em: https://www.dgterritorio.gov.pt/recart/CONSTRU%C3%87%C3%95ES/Edificio.html#page_valorUtilizacaoAtual
 
A representação deverá respeitar as regras para cada nível de detalhe, de acordo com as regras globais e específicas. Ver: https://www.dgterritorio.gov.pt/recart/imagens/eEdificio1.jpg

Obrigado pelo feedback.

**Comment by CatarinaGaspar** [comment]
Boa Tarde, 

Obrigado pelo esclarecimento.

Bom trabalho,

**Comment by CatarinaGaspar** [comment]
Boa Tarde, 

Analisando melhor esta situação, fica estranho colocar estas passagens como edificios associados à residência. Pois na realidade não são edificios.

Não faria mais sentido colocar estas passagens no tema de construções poligonais? Como se fez com as escadas?

Obrigado,

**Comment by aserronha** [comment]
Existem "passagens aéreas" semelhantes que não têm escadaria...

**Comment by MartaValenteMuni** [comment]
Boa tarde,

Em dezembro de 2020 esta mesma questão foi colocada e foi respondido que as passagens não seriam representadas.

![image](https://user-images.githubusercontent.com/60008526/221236101-98d781b0-a48b-48c7-be1b-347091c6a065.png)

Compreendemos que tenha sido alterada desde então a designação de "Associado à Residência", tendo sido acrescentado "ou similares". Estas passagens correspondem a "similares"? Que outros exemplos haverá também? Sendo estas passagens consideradas um edifício, faz sentido o atributo altura?

**Comment by CatarinaGaspar** [comment]
> Existem "passagens aéreas" semelhantes que não têm escadaria...

Certo, estava a comparar com à situação das escadas. Que anteriormente faziam parte do tema edificio (associado à residencia) e depois passaram para o tema das construções poligonais. Não queria dizer que eram para ficar classificadas como "escadas".
Eventualmente teria que se criar um valor apropriado.

**Comment by aserronha** [comment]
Boa tarde,

De acordo com que foi apresentado e como a resposta que demos anteriormente não foi documentada no github (deve ter sido alguma resposta a um e-mail em particular) decidimos que nas especificações técnicas em vigor pode não se considerar como "ou similares" essas "passagens aéreas" e é preferível (para já) manter a coerência de não serem representadas.

Está mesmo quase a ser concluída a nova versão das Especificações Técnicas e vamos avaliar esta questão.

Obrigado pelo feedback.

**Comment by aserronha** [comment]
Boa tarde.

Esta questão foi avaliada e este elemento continua a não estar previsto nestas ET v2.0.
